examples/s2s-api/oxxo_payment.php

```diff
@@ -90,17 +91,24 @@
     $response = $payment->postRequest();
 
     echo "\n\nRESULT BELOW\n";
-    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_VERIFICATION_REQUIRED) {
+    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_OK) {
         echo "The transactions was successfully processed. Vendo's Transaction ID is: " . $response->getTransactionDetails()->getId();
```

Comment:
@pwolowiec-zz you removed the text related to oxxo and used text for credit card payments instead.

Reply:
Add a php comment to the line where the PDF is retrieved "You must send this PDF to the end user" or something like that.

Reply:
This delete is correct; we won't send them pdf and barcodes. Just verification url for their customers. When customer click on that link we will generate OXXO payment for him and display our confirmation page with barcode and PDF